rules-reviewed/openshift/jni-native-code.windup.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0"?>
+<ruleset id="jni-native-code"
+    xmlns="http://windup.jboss.org/schema/jboss-ruleset"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://windup.jboss.org/schema/jboss-ruleset http://windup.jboss.org/schema/jboss-ruleset/windup-jboss-ruleset.xsd">
+    <metadata>
+        <description>
+            This is a ruleset for detecting JNI native code usage usage when migrating an application to a cloud environment.
+        </description>
+        <dependencies>
+          <addon id="org.jboss.windup.rules,windup-rules-javaee,3.0.0.Final"/>
+          <addon id="org.jboss.windup.rules,windup-rules-java,3.0.0.Final"/>
+        </dependencies>
+        <targetTechnology id="cloud-readiness"/>
+        <tag>jni</tag>
+    </metadata>
+    <rules>
```

Comment:
I don't understand.
This rule already detects the usage of JNA: <javaclass references="com.sun.jna.{*}">